assets/src/scss/_ad-styles.scss

```diff
@@ -0,0 +1,220 @@
+/* Taken from https://github.com/readthedocs/ethical-ad-client/blob/master/styles.scss */
```

Comment:
Seems like it would be better for us to publish these as part of the client, and link them from there. That would keep us up to date with changes.

Reply:
Currently, the ad server does not rely directly on the client. Are you proposing we do that? Or are you just proposing that we build the styles from the client into the CSS for the ad server by making the client a dev dependency?

Reply:
Not sure -- we can deal with it later, but copying & pasting them here feels like it will get out of date, but we probably have a while before that's an issue, and we can deal with it the.

Reply:
Cool. I removed it and imported it from node. However, we will need to manage the version we link against. Should be better than the first pass though.